docs/data-sources/fabric_cloud_routers.md

```diff
@@ -1,9 +1,5 @@
 ---
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "equinix_fabric_cloud_routers Data Source - terraform-provider-equinix"
 subcategory: "Fabric"
-description: |-
```

Comment:
Can we add description into the resource definition to get matching markdown results? This is a small description, but we could miss some handy documentation links or greater details if they were specified on other resources (I'll keep an eye out in this review).

I presume the default page_title is as good or better.

Reply:
The descriptions are in the body, they are not being rendered in the markdown frontmatter.  

https://github.com/equinix/terraform-provider-equinix/pull/764/files?diff=split&w=1#diff-329a375d2de207d4b7529b502ca8930f5636802bc6aa8934c848ed2972a7260aR32-R36

Is this intended? The first line of non-title markdown body text ends up getting the same treatment as a description frontmatter so it would be redundant to capture it as front-matter?

Reply:
The `description` frontmatter is not used by the registry: https://developer.hashicorp.com/terraform/registry/providers/docs#yaml-frontmatter

The `page_title` is used, but based on current page titles in the registry, the value we are specifying in data source and resource pages is equivalent to not specifying that attribute at all.